### ISSUE TITLE
API read failures sometimes return 200 with empty body

### ISSUE THREAD

**Issue by adstage-david** [untangled-web/untangled-server#10]
In my example based on the cookbook, clicking the new button causes the failure state I've seen in our production app:

https://github.com/untangled-web/untangled-cookbook/pull/5

Essentially if some random function throws an exception during a read error, it's not properly caught and handled.

**Comment by awkay** [comment]
@egracer, was this due to the Om parser swallowing them?

**Comment by egracer** [comment]
@awkay I don't believe so, because the default server read throws an error that is successfully propagated to the client.
